tests/layers/test_signal.py

```diff
@@ -241,3 +241,50 @@ def test_mel_spectrogram(
     )(x).numpy()
     # These tolerances make look large, but the values we're comparing are in the 10e2-10e4 range.
     assert np.allclose(librosa_spec, rgp_spec, atol=1e-2, rtol=0)
+
+def test_magnitude() -> None:
+    x = np.random.normal(0, 1, 1024)
+    x_stft = librosa.stft(
+        x,
+        n_fft=256,
+        hop_length=256,
+        win_length=256,
+        center=True,
+    ).T
+
+    np_magnitude = np.abs(x_stft)
+    layer_magnitude = signal.Magnitude()(tf.expand_dims(x_stft, 0)).numpy()
+
+    assert np.allclose(np_magnitude, layer_magnitude, atol=1e-3, rtol=0)
+
+
+def test_phase() -> None:
```

Comment:
Same question as before, could you add a few obvious cases as well? For example `1+j`, `0`, j`, `-j`.

Reply:
Add test cases:
* Random
* [[0],[1],[2],[1j],[2j],],
* [[1+2j],[3-4j],[-5+6j],[-7-8j],[9+10j],],